docs/src/routes/docs/[...1]overview/[...2]contribution-guide.md

```diff
@@ -2,54 +2,74 @@
+We use both semantic and `alpha` versioning for publishing and testing packages prior to official release.  If you do not see an `-alpha.x` tag on the version of the package you made changes in, you will need to bump the version AND add an alpha flag followed by an alpha version number starting at 1 (ex. `-alpha.1`).
 
-We use both semantic and `alpha` versioning for publishing and testing packages prior to official release.  This means if you do not see an `-alpha.x` tag on the version of the package you made changes in you will need to not only bump the version but also add an alpha flag followed by an alpha version number starting at 1 (ex. `-alpha.1`).
+After making changes within a web3-onboard package, bump the version of the specific package:
 
 Example : Adding a new injected wallet to the `injected` package -
 [Injected Package.json](https://github.com/blocknative/web3-onboard/blob/develop/packages/injected/package.json#L3) changes : `"version": "2.2.4",` --> `"version": "2.3.0-alpha.1",`
 
 Example 2 : Bug fix within the `core` package -
 [Core package.json](https://github.com/blocknative/web3-onboard/blob/8531a73d69365f7d584320f1c4b97a5d90f1c34e/packages/core/package.json#L3) changes: `"version": "2.9.1-alpha.1",` --> `"version": "2.9.1-alpha.2",`
 
+## [How can I add a new wallet? 💳](#new-wallet)
 
-### How can I get a new wallet added?
+To add a new wallet to the official Web3 Onboard packages and repo, create a [pull request](#pull-requests-🗂️). Web3 Onboard does not require a wallet to be a part of the main code, so a separate wallet module can be created without any changes to the Web3Onboard codebase. Your pull request must include a README for the package.  
 
-Web3 Onboard does not require a wallet to be a part of the main codebase to work, so a wallet module can be created and used for your project without needing anything to happen within the Web3Onboard codebase. If you would like the wallet to be part of the official Web3 Onboard packages and repo, then create a pull request, and make sure to add any documentation updates by creating a docs pull request.
+Please also add documentation updates by creating a separate [docs pull request](#documentation-contributions-📄).
 
-If the wallet you are adding is an “injected” wallet (browser extension, mobile dapp browser wallet), you can add a wallet to the injected wallets module. [See here for an example of an injected wallets pull request.](https://github.com/blocknative/web3-onboard/pull/1177/files) You should also add the wallet to the [natively supported injected wallets list](https://onboard.blocknative.com/docs/packages/injected#injected-wallets-supported-natively).
+### Adding Injected Wallets
```

Comment:
Might be helpful to reiterate that the PR checklist must be complete prior to review in a few spots throughout this